pkg/loader/compose/compose.go

```diff
@@ -358,6 +358,12 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 			}
 		}
 
+		if len(serviceConfig.Port) == 0 {
+			if composeServiceConfig.Labels["kompose.service.type"] == "NodePort" || composeServiceConfig.Labels["kompose.service.type"] == "LoadBalancer" {
```

Comment:
Why are there nested if's? 

Shouldn't it be: `if len(serviceConfig.Port) == 0 &&  composeServiceConfig.Labels["kompose.service.type"] == "NodePort" || composeServiceConfig.Labels["kompose.service.type"] == "LoadBalancer"`